test/shared/data/index.ts

```diff
@@ -1,5 +1,6 @@
 import path from "path"
 import fs from "fs-extra"
+import {ReadableStream} from "web-streams-polyfill/ponyfill"
```

Comment:
I added a web-file-polyfill for this in my own tests. You'll see it in the upcoming PR.